src/Compilers/Core/Portable/Emit/EditAndContinue/EncVariableSlotAllocator.cs

```diff
@@ -249,7 +249,7 @@ private bool TryGetPreviousLambdaSyntaxOffset(SyntaxNode lambdaOrLambdaBodySynta
         {
             // Syntax map contains mapping for lambdas, but not their bodies. 
             // Map the lambda first and then determine the corresponding body.
-            var currentLambdaSyntax = isLambdaBody ? lambdaOrLambdaBodySyntax.GetLambda() : lambdaOrLambdaBodySyntax;
+            var currentLambdaSyntax = isLambdaBody ? GetLambda(lambdaOrLambdaBodySyntax) : lambdaOrLambdaBodySyntax;
```

Comment:
@tmat This was the only call to .GetLambda.  So it seems like GetLambda was very tied to the EncVariableSlotAllocator